main.cpp

```diff
@@ -1 +1,307 @@
+#include <iostream> //this is only for cout and tolower
+#include <stdlib.h>
+#include <string.h>
 
+
+// Initializes list
+// void StringListInit(char*** list); 
+void StringListInit(char*** list)
+{
+    if (*list) return;
+    *list = (char**)calloc(2, sizeof(char*));   
+}
+
+// Destroy list and set pointer to NULL.
+// void StringListDestroy(char*** list);
+void StringListDestroy(char*** list)
+{
+    if (!*list) return;
+    char** current(*list), **previous(NULL);
```

Comment:
Please use assignment operator to initialize pointer. C language does not allow it.